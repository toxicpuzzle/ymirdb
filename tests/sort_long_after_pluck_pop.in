SET a 99 88 10 69 28 39 4 2 1 2 0 
SORT a
GET a
PLUCK a 3
POP a
GET a
PUSH a -2 -3 -100 -21 22
GET a
SORT a 
GET a
bye