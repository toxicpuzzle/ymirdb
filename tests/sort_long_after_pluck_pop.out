> ok

> ok

> [0 1 2 2 4 10 28 39 69 88 99]

> 2

> 0

> [1 2 4 10 28 39 69 88 99]

> ok

> [22 -21 -100 -3 -2 1 2 4 10 28 39 69 88 99]

> ok

> [-100 -21 -3 -2 1 2 4 10 22 28 39 69 88 99]

> bye
